src/components/ContentHighlights/data/tests/hooks.test.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/jsx-filename-extension */
+import { useState } from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ContentHighlightsContext, initialStateValue, testCourseAggregation } from '../../../../data/tests/ContentHighlightsTestData';
+import { useContentHighlightsContext } from '../hooks';
+
+const TestComponent = () => {
```

Comment:
nit: Is there a more explicit component name that might better describe its purpose than `TestComponent`?